test/importers/fixtures/complex.spec.md

```diff
@@ -1,22 +1,22 @@
 # How Sample Performance Management Improved in 2021
 
-![](https://www.sample.com/blog/wp-content/uploads/How-Sample-Performance-Management-Improved-in-2021_1920x733-scaled.jpg)
+![][image0]
```

Comment:
If we import as markdown, what do we do with this ? No way to use the markdown output.

Reply:
yes. this is default markdown syntax:

# watch this!
![][image0]


[image0]: http://dummyimage.com/300.png